tests/test_jsonapi/test_fetching_data.py

```diff
@@ -377,6 +377,25 @@ def test_relationship_links(self):
         assert links['self'].endswith('/article/1/relationships/author')
         assert links['related'].endswith('/article/1/author')
 
+    def test_explicitly_set_primary_key_in_links(self):
+        """Should use the primary_key explicitly set when generating links"""
+        self.manager.create_api(self.Article, primary_key='title')
```

Comment:
This line should be moved after the `session.commit()` to match the other tests.